Mathlib/LinearAlgebra/AffineSpace/ContinuousAffineEquiv.lean

```diff
@@ -112,6 +114,29 @@ theorem ext {e e' : P₁ ≃ᵃL[k] P₂} (h : ∀ x, e x = e' x) : e = e' :=
 protected theorem continuous (e : P₁ ≃ᵃL[k] P₂) : Continuous e :=
   e.2
 
+/-- A continuous affine equivalence is a continuous affine map. -/
+def toContinuousAffineMap (e : P₁ ≃ᵃL[k] P₂) : P₁ →ᴬ[k] P₂ where
+  __ := e
+  cont := e.continuous_toFun
+
+@[simp]
+lemma coe_toContinuousAffineMap (e : P₁ ≃ᵃL[k] P₂) : ⇑e.toContinuousAffineMap = e :=
+  rfl
+
+lemma toContinuousAffineMap_injective :
+    Function.Injective (toContinuousAffineMap : (P₁ ≃ᵃL[k] P₂) → (P₁ →ᴬ[k] P₂)) := by
+  intro e e' h
+  ext p
+  simp_rw [← coe_toContinuousAffineMap, h]
+
+lemma toContinuousAffineMap_toAffineMap (e : P₁ ≃ᵃL[k] P₂) :
+    e.toContinuousAffineMap.toAffineMap = e.toAffineEquiv.toAffineMap :=
+  rfl
+
+lemma toContinuousAffineMap_toContinuousMap (e : P₁ ≃ᵃL[k] P₂) :
+    e.toContinuousAffineMap.toContinuousMap = toContinuousMap e.toHomeomorph :=
+  rfl
+
```

Comment:
Another good lemma to have is `id.toContinuousAffineMap = ContinuousAffineMap.id`, and similar for `.trans`